runtime/autoload/man.vim

```diff
@@ -406,4 +410,32 @@ function! man#init_pager() abort
   endif
 endfunction
 
+let s:goto_via_filename = v:false
+
+function! man#goto_tag(pattern, flags, info) abort
+  " currently no support for section completion
+  let sect = ""
+
+  if s:goto_via_filename
+    let candidates = s:get_paths(sect, a:pattern)
+    return map(candidates, {
+    \ _, path -> {
+    \     'name': s:extract_sect_and_name_path(path)[1], " this leaves us on the raw man-page file
+    \     'filename': path,
+    \     'cmd': '1'
+    \   }
+    \ })
+  else
+    let candidates = s:complete(sect, sect, a:pattern)
+    let dummyfile = expand("%")
+    return map(candidates, {
+    \   _, name -> {
+    \     "name": name,
+    \     "filename": dummyfile,
+    \     "cmd": "Man " . name " we can't execute :Man like this, sandbox prevents it
```

Comment:
I assume that restriction is for 'modeline', but now that `modelineexpr=0` is the default,  we could remove the sandbox restriction here. 

Perhaps 'modelineexpr' should be permanent/non-optional.

Reply:
I've gone with the other approach (which feels less hacky without this dummy filename stuff), but out of interest, by removing the sandbox restriction, do you mean making a change to the C code to disable sandbox execution for `'tagfunc'` ?

Reply:
> , by removing the sandbox restriction, do you mean making a change to the C code to disable sandbox execution for 'tagfunc' ?

Yes